Tools/jit/build.py

```diff
@@ -0,0 +1,22 @@
+"""Build an experimental just-in-time compiler for CPython."""
+
+import argparse
+import pathlib
+
+import _targets
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
Hm, not sure when this would be needed though. The hash depends on `Python/executor_cases.c.h`, which covers any relevant changes in the uops. And the `jit_stencils.h` doesn't depend on `jit.c`... the dependency is the other way around.

I guess maybe we aren't covering the case where one of the included headers in `template.c` changes... I'll go ahead and add a `--force` option then, since that's not too difficult.